wlauto/common/android/workload.py

```diff
@@ -267,7 +303,12 @@ def _grant_requested_permissions(self):
                 break
 
         for permission in permissions:
-            self.device.execute("pm grant {} {}".format(self.package, permission))
+            # "Normal" Permisions are automatically granted and cannot be changed
+            for normal in ANDROID_NORMAL_PERMISSIONS:
+                if normal in permission:
+                    break
+            else:
+                self.device.execute("pm grant {} {}".format(self.package, permission))
```

Comment:
instead of the inner `for` loop:

    permission_name = permission.rsplit('.', 1)[1]
    if permission_name not in ANDROID_NORMAL_PERMISSIONS:
        self.device.execute("pm grant {} {}".format(self.package, permission))